src/ARCValidationPackages/AVPRAPI.fs

```diff
@@ -0,0 +1,34 @@
+namespace ARCValidationPackages
+
+open AVPRClient
+open ARCValidationPackages
+open AVPRIndex
+open System
+open System.Net.Http
+
+type AVPRAPI () =
+    member private this.BaseUri = Uri("https://avpr.nfdi4plants.org")
```

Comment:
I'd prefer these to be refactored into Globals namespace. I am also quite sure that the default base url is already set to exactly this url when you call the client constructor